LondonBritishLibrary/orient/BLorient799.xml

```diff
@@ -60,61 +124,124 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                      <persName ref="PRS3501Demetros">Dǝmeṭros</persName>, <persName ref="PRS3692egwalas">ʾƎgʷāla Ṣǝyon or Gʷālu</persName>,
+                      <persName ref="PRS4661Gigar">Gigār</persName>, <persName ref="PRS4264GabraKr">Gabra Krǝstos</persName> and <persName ref="PRS8344sahlaDe">Sāhlu</persName>. The following
+                    are the most important of them.</note>
+                          <list>
+                          <item xml:id="a1">
+                            <locus target="#215r"/>
+                            <desc type="LandGrant">Deed of gift by <persName ref="PRS2377Bakaffa">Bakāffā</persName> to the priests of the church of Raphael.</desc>
+                            <q xml:lang="gez">ነዋ፡ ወሀብነ፡ ንሕነ፡ ንጉሠ፡ ነገሥት፡ በካፋ። ለካህናት፡ ዘቅዱስ፡ ሩፋኤል፡ ከመ፡ ይኩነነ፡ ለመድኃኒተ፡ ሥጋ፡ ወነፍስ። ፬፻፵፯ምድር፡
+                            በፍንጥሮ፡ ወአባጌ፡ በባሕር፡ ሽሽ። በኰሰን፡ ፵፬ምድር። በፈንታይ፡ በወይዘሮ፡ ወለተ፡ ሐዋርያትና፡ በወይዘሮ፡ ጽዮን፡ ሞገሳ፡ እጅ፡ ላ፲፪ጫን፡ ማድጋ፡ በጎደለው፡
+                          ያፄ፡ ቆሎ፡ ምድር፡ ፸ወ፯፡ ደብተራ፡ የተሰራ። በወይዘሮ፡ ባዩሽ፡ ወንድም፡ በውስጤ፡ እጅ፡ በነበረው።። በአለዋ፡ በጅአግሬው፡ በዱሪ፡ ማሞ፡ እጅ፡ በነበረው፡ ፲ወ፪፡ ደብተራ፡
+                         ዘንተ፡ ሥርዓተ፡ ዘነሰተ፡ እመኒ፡ ንጉሥ፡ ወእመሂ፡ ንግሥት፡ እመኒ፡ መኰንን፡ ወእመሂ፡ ሕዝባዊ፡ በሥልጣነ፡ ጴጥሮስ፡ ወጳውሎስ፡ ወበ፲ወ፪ሐዋርያት፡ ውጉዘ፡ ለይኩን።</q>
+                            <date notBefore="1721" notAfter="1730"/>
+                          </item>
+
+                          <item xml:id="a2">
+                          <locus target="#87b #88r"/>
+                          <desc type="MixedNote">List of priests in the above church.</desc>
+                          <q xml:lang="gez">ኈልቈ፡ ካህናት፡ ዘቅዱስ፡ ሩፋኤል፡ ዘአስተጋብኦሙ፡ ነጉሠ፡ ነገሥት፡ በካፋ።</q>
+                          <note>They are arranged into right, f. 87v, and left, f. 88r, according to the months of the year.</note>
+                          </item>
+
+<item xml:id="a3">
+<locus target="#3r"/>
+<desc type="DonationNote">Deed of gift of <persName ref="PRS5617IyasuII">Iyāsu II</persName> and <persName ref="PRS2682BerhanM">Walatta Giyorgis</persName>
+ to the convent of <placeName ref="INS0330DDQ">Dabra Ḍaḥāy</placeName> near <placeName ref="LOC3577Gondar">Gondar</placeName>.</desc>
+<q xml:lang="gez">ነዋ፡ ወሀብነ፡ ንሕነ፡ አያሱ፡<sic/> ዘስመ፡ መንግሥትነ፡ ብርሃን፡ ሰገድ፡ ወእመነ፡ ንግሥት፡ ወለተ፡ ጊዮርጊስ፡ ብርሃን፡ ሞገሳ፡ ለቤተ፡ ክርስቲያን፡ ቍስቋም፡ እንተ፡ ሕንጽት፡
+በምድረ፡ ላይ፡ ጎደር፡<sic/> ዘተሰምየት፡ ደብረ፡ ፀሐይ፡ አልባሰ፡ ዘሐሪር፡ ወዘርቤታተ፡ ወማዕጠንታተ፡ ወአክሊላተ፡ ዘወርቅ፡ ወዘብሩር፡ ፃሕለ፡ ወጽዋዓ፡ ዘብሩር፡ ወመጻሕፍተ፡ ዘብሉይ፡ ወዘሐዲስ፡
+ወኵሎ፡ ንዋያተ፡ ቅድሳት፡ ከመ፡ ይግበሩ፡ ባሙ፡<sic/> መስዋዕተ፡ ለእግዚአብሔር፡ ለመድኃኒተ፡ ንፍስነ፡ ወለካህናቲሃ፡ ወሀብናሆሙ፡ ሀገረ፡ ዘይሰመይ፡ ባጀና፡ ቀዳሚ፡ ዘነበረ፡ ቦቱ፡ ጋላ፡ ወድኅረ፡
+ዘተሠርዓ፡ ቦቱ፡ ዘዌ፡ ከመ፡ ይኩን፡ ለሲሳዮሙ፡ ወለቁርባንሂ፡ ወሀብነ፡ ሀገረ፡ ዘነበረ፡ በእደ፡ ወይዘሮ፡ መስለ፡ ንትያ፡ ዘስሙ፡ ጉንትር፡ ወዘተዝካረ፡ ንጉሥኒ፡ ሀገረ፡ ዘይሰመይ፡ ከች፡ ወለመካንሂ፡
+ወለጥነ፡ መካነ፡ ዘይሰመይ፡ ቆፋሮች፡ በቅድመ፡ አቡነ፡ <persName ref="PRS6072Krestodo">ክርስተደወሉ፡</persName> ጳጳስ፡ ዘኢትዮጵያ፡ ወበቅድመ፡
+አቡነ፡ <persName ref="PRS12300TaklaHaymanot">ተክለ፡ ሃይማኖት፡</persName> መምኅረ፡ ዘደብረ፡ ሊባኖስ፡ መበቅድመ፡<sic/> ዓቃቤ፡ ሰዓት፡ ወሊቃው፡<sic/> አብያተ፡ ክርስቲያናት፡<gap reason="omitted" resp="PRS10171Wright"></gap></q>
```

Comment:
I think it is better to encode this not with an omission, but using a seg. Thus pointing that the text is just an incipit of an addition.